packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1517,6 +1517,19 @@ class Positioned extends ParentDataWidget<Stack> {
        bottom = null,
        super(key: key, child: child);
 
+  /// Creates a Positioned object with [left], [top], [right], and [bottom] set
+  /// to 0.0.
+  Positioned.fixedToEdge({
```

Comment:
You could provide Positioned.fromInsets() - without a default value for insets - and Positioned.cover() that was defined as Positioned.fromInsets with insets: EdgeInsets.zero